docs/dev-guide.md

```diff
@@ -230,32 +185,14 @@ In general we prefer simplicity. We standardize on JavaScript (Node.js) and SQL
 From JavaScript you can then do:
 
 ```javascript
-var sqldb = require('@prairielearn/postgres');
-var sql = sqldb.loadSqlEquiv(__filename); // from filename.js will load filename.sql
+import { loadSqlEquiv, queryRow } from '@prairielearn/postgres';
 
-// run the entire contents of the SQL file
-sqldb.query(sql.all, params, ...);
+import { QuestionSchema } from './lib/db-types.js';
 
-// run just one query block from the SQL file
-sqldb.query(sql.select_question, params, ...);
-```
+const sql = loadSqlEquiv(import.meta.url);
 
-- The layout of the SQL code should generally have each list in separate indented blocks, like:
-
-  ```sql
-  SELECT
-    ft.col1,
-    ft.col2 AS renamed_col,
-    st.col1
-  FROM
-    first_table AS ft
-    JOIN second_table AS st ON (st.first_table_id = ft.id)
-  WHERE
-    ft.col3 = select3
-    AND st.col2 = select2
-  ORDER BY
-    ft.col1;
-  ```
+const rows = await queryRow(sql.select_question, { question_id: 45 }, QuestionSchema);
```

Comment:
Either change to `queryRows` with appropriate arguments, or rename variable to `question` or `row`.
```suggestion
const question = await queryRow(sql.select_question, { question_id: 45 }, QuestionSchema);
```

Reply:
Fixed in 7ec3678.